### PR TITLE
Add basic structure for site

### LINKED ISSUE #6: Add initial files
This will include an index.html, css and js files.

### PR DESCRIPTION
**Pull request by hemangsk**
Fixes #6

### CODE REVIEW

File: partials/tabs/projects.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+<section>
+
+<div class="container">
+    <div class="row showcase">
+    <div ng-click="lc.showProject(project)" ng-repeat="project in projectList | filter: searchText" class="card card-main showcase-card">
+        <div class="card-title name-bear">{{ project.name }}</div>
+        <div class="card-content">{{ project.desc | format_desc }}</div>
+        <div class="card-action">
+            <span ng-repeat="lang in project.mentors | limitTo:4">{{lang}} </span>
```
Comment (sils) [inline partials/tabs/projects.html]: lang?you mean mentor

File: partials/tabs/projects.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        </div>
+    </div>
+
+    <div class="" ng-repeat="sda in lc.projectList">{{sda}}</div>
```
Comment (sils) [inline partials/tabs/projects.html]: what is sda? Not obvious for sure.

File: partials/tabs/projects.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        </div>
+    </div>
+
+</div>
+
+<section>
+
+<div class="container">
+    <div class="row showcase">
+    <div ng-click="lc.showProject(project)" ng-repeat="project in projectList | filter: searchText" class="card card-main showcase-card">
```
Comment (sils) [inline partials/tabs/projects.html]: this should be indented

File: resources/css/style.css [hunk 4]
```diff
@@ -0,0 +1,511 @@
+
```
Comment (sils) [inline resources/css/style.css]: redundant newline

File: resources/css/style.css [hunk 5]
```diff
@@ -0,0 +1,511 @@
...
+  font-family: Roboto;
+  font-size: 1.6em;
+  text-align: center;
+  font-weight: 300;
+  text-transform: none !important;
+}
+.term-install-card, pre{
+  background-color: #2b2b2b;
+  color: #fff;
+  font-family: Monaco, Menlo, "Ubuntu Mono", monospace;
```
Comment (sils) [inline resources/css/style.css]: not roboto?

File: resources/css/style.css [hunk 6]
```diff
@@ -0,0 +1,511 @@
...
+  letter-spacing: 0.1em;
+}
+.description{
+  font-family: Roboto;
+  font-size: 1.6em;
+  text-align: center;
+  font-weight: 300;
+  text-transform: none !important;
+}
+.term-install-card, pre{
```
Comment (sils) [inline resources/css/style.css]: actually are those even used?

Comment (hemangsk) [inline resources/css/style.css]: No no they are not being used

**Comment by sils** [comment]
92cdd5c is ready

**Pull request merged**